lib/index.js

```diff
@@ -72,15 +73,53 @@ function qRouter(spec) {
         };
     }
 
-    function setProgress(sectionId) {
+    function restoreAnswer(sectionId) {
+        const answerToRestore = q.retractedAnswers[sectionId];
+
+        if (answerToRestore !== undefined) {
+            q.answers[sectionId] = answerToRestore;
+            delete q.retractedAnswers[sectionId];
+        }
+
+        return answerToRestore;
+    }
+
+    function retractAnswers(sectionIds) {
+        const retractedAnswers = [];
```

Comment:
What is the purpose for maintaining & returning this array?

Reply:
To keep the api consistent, all methods return the result of their action. In this particular instance it's not currently used.